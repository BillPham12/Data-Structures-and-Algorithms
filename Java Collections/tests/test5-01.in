1
123
hello
mey i have
123
1
my name is gosu