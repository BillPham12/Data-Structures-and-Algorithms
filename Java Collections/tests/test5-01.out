1
123
hello
mey i have
my name is gosu
